pytorch_lightning/core/lightning.py

```diff
@@ -1964,8 +1964,14 @@ def model_size(self) -> float:
         """
         The model's size in megabytes. The computation includes everything in the
         :meth:`~torch.nn.Module.state_dict`, i.e., by default the parameteters and buffers.
+
+        .. deprecated::v1.4
+            Will be removed in v1.6.0
         """
-        # todo: think about better way without need to dump model to drive
+        rank_zero_deprecation(
+            'The `LightningModule.model_size` property was deprecated in v1.4 and will be removed in v1.6.'
```

Comment:
I miss what is the alternative here?

Reply:
@Borda the alternative would be to move this function to an utility function, I am just not sure to include that within this PR or a new one.

Reply:
@roshikouhai we can pull the changes from https://github.com/PyTorchLightning/pytorch-lightning/pull/8500/ into this PR to make this deprecation clearer. 

We can also update existing callers of this property to the new utility